src/index.html

```diff
@@ -248,7 +250,7 @@ <h1 class="c-slide__title col-md-6">Scale up your team with experienced develope
                 </section>
                 
                 <div class="l-bg">
-                    <img src="./img/c-loop-bg.jpg" alt="background-loop" class="c-loop-bg l-loop-bg">
```

Comment:
Maybe we could get rid off js-lazy-img and just select IMG with data-src from jQuery